### DISCUSSION TITLE
EWW Widgets on Login Manager screen

### DISCUSSION THREAD

**Discussion by mhqqa** [elkowar/eww#485] [General]
I installed xdm and was wondering if I can add a bar with power buttons and a clock, but eww panics - crates/eww/src/main.rs:235:73, something about $HOME not being present? env::var("HOME")?. 
Is there a workaround or would I run into other issues? I just like the idea of light login managers with eww flare and xdm seemed easy to configure. (maybe even xlogin interface?)
disclaimer - I'm novice *nix user and about 2 weeks into rust, so can't do much else for know to figure it out, but was wondering if something like this would be possible in the future.

Thanks in advance

**Comment by elkowar** [comment]
haha, yea, you're not logged in whilst in the display manager -- thus $HOME does not exist yet. I'm not sure if it would work otherwise tho, as I'm pretty sure that display managers would be quite restrictive (i.e. not implementing an actual WM that eww could talk to) -- but it's worth a shot I guess. You should be able to just run the eww daemon and set `$HOME` manually, i.e. `HOME="/home/you/" eww daemon`
